### ISSUE TITLE
C'Thin doesn't take the correct vulnerability damage in phase 2.

### ISSUE THREAD

**Issue by Kazzekk** [Duskhaven/Bug-Reports#1884]
In P2 of C'Thun, once you kill the eye tentacles in his stomach, he doesn't obtain the vulnerability debuff properly. He takes normal damage, and only for 30 seconds. After that 30 seconds is up, he doesn't take damage anymore like the start of Phase 2. This makes the fight mathematically impossible, as to complete this you would have to keep people up for hours.

Tentacles also do not get stunned and they do not stop spawning during the phase that C'Thun is supposed to be vulnerable.
Also, it would appear that the portal to teleport you to C'Thun does not stick around. There was already a bug-fix that was marked "Completed" regarding this, however I'm assuming this was done via a fresh ID. The runs that I attempted C'Thun on with my guild were runs 2-3 days after the initial boss kills, and the NPCs to take you to C'Thun after downing the Emperors were not there or not working.

TLDR: Vulnerability doesn't apply the 300% damage debuff, instead he takes normal damage, and afterward he goes back to taking no damage until you kill stomach tentacles again. This isn't sustainable at all and is a bug. And the tentacles do not get stunned and they do not stop spawning during the phase that C'Thun is supposed to be vulnerable.
